destalinator.py

```diff
@@ -75,6 +72,13 @@ def debug(self, message):
         if self.output_debug_to_slack_flag:
             self.log(message)
 
+    def flush_channel_cache(self, channel_name):
+        """Flush all internal caches for this channel name."""
```

Comment:
This is to reduce the memory footprint? Were there reports of running out of memory?